Index: assignment 3.py
===================================================================
diff --git a/assignment 3.py b/assignment 3.py
new file mode 100644
--- /dev/null	(date 1759206922391)
+++ b/assignment 3.py	(date 1759206922391)
@@ -0,0 +1,42 @@
+#bai 1
+a = int(input("nhap 1 so nguyen: "))
+b = str(a)[::-1]
+print(b)
+
+#bai 2
+a = int(input("nhap so nguyen 1: "))
+b = int(input("nhap so nguyen 2: "))
+print("truoc khi doi: a =", a, ", b =", b)
+
+a = a ^ b
+b = a ^ b
+a = a ^ b
+
+print("sau khi doi: a =", a, ", b =", b)
+
+#bai 3
+a = int(input("nhap 1 so nguyen: "))
+if a > 0 and (a & (a - 1)) == 0:
+    print("false")
+else:
+    print("true")
+
+import math
+#bai 4
+x = int(input("nhap so nguyen 1: "))
+y = int(input("nhap so nguyen 2: "))
+print(math.floor(x//y))
+
+
+#bai 5
+x = int(input("nhap so nguyen 1: "))
+y = int(input("nhap so nguyen 2: "))
+print(math.ceil(x//y))
+
+#bai 6
+d = int(input("nhap 1 so nguyen: "))
+if d % 2 == 0:
+    print("even")
+else:
+    print("odd")
+
diff --git a/bai-tap-tuan-3.py b/bai-tap-tuan-3.py
deleted file mode 100644
